composer.json

```
@@ -17,7 +17,12 @@
         "geoip2/geoip2": "^2.9",
         "symfony/stopwatch": "^6.2",
         "symfony/form": "^6.2",
-        "lcobucci/jwt": "^4.0"
+        "lcobucci/jwt": "^4.0",
+        "matomo/device-detector": "^6.0",
+        "pimcore/admin-ui-classic-bundle": "1.x-dev",
```

Since we have `"minimum-stability": "dev",` set it is not necessary to require `1.x-dev` right?

```suggestion
        "pimcore/admin-ui-classic-bundle": "^1.0",
```

---

Yes, right. Thanks